spark/pom.xml

```diff
@@ -192,6 +192,15 @@
 
     <profiles>
 
+        <profile>
+            <id>spark-2.4</id>
```

Comment:
Hm, I thought this profile is not meant to be used for building. I referred https://github.com/apache/zeppelin/pull/2880.

Reply:
BTW, I am verifying this in my personal Travis CI as well (https://travis-ci.org/HyukjinKwon/zeppelin)

Reply:
@HyukjinKwon Could you update `.travis.yml` add test matrix for spark 2.4